api/v1beta1/constants.go

```diff
@@ -0,0 +1,5 @@
+package v1beta1
+
+const (
+	GoldenImagesNSname = "kubevirt-os-images"
```

Comment:
Is it a good idea to export a hardcoded namespaces as part of API?
Is there a reason why golden images cannot be in a different namespace?

Reply:
I'm not sure why that has been hardcoded in the past, but it is.
If we want to change that, I'd prefer to make that decision at another place and change it then.
Regarding the export, I probably wouldn't hardcode it in the first place, but that's what it is. Having it here in the API it's at least consumable by anything that needs to know the namespace.